calculator.js

```diff
@@ -1,27 +1,90 @@
-
 const exampleAdditionInput = {
   num1: 3,
   num2: 5,
   operation: 'add',
 }
 
+function getValidOperator(operator) {
+  const operators = [ 'add', '+', 'subtract', '-', 'multiply', '*', 'divide', '/']
+
+  if (operators.includes(operator)){
+    return operator;
+  }
+  return undefined;
+}
+
+function getValidNums(number) {
+  if (isNaN) {
+    return parseInt(number);
+  }
+  return undefined;
+}
+
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
 
   // Questions to ask and answer:
   // What is promptInput?
   // What data type? What does it hold? What does it represent?
   // How do we read values from it? What syntax?
   // How can we use it?
   // Can we call our existing functions now, inside of this function?
+
+  let num1 = getValidNums(promptInput.num1);
+    if (num1 == null) {
+      console.log(`Invalid input: ${promptInput.num1}`);
+      return;
+    };
+
+  let num2 = getValidNums(promptInput.num2);
+    if (num2 == null) {
+      console.log(`Invalid input: ${promptInput.num2}`);
+      return;
+    };
+
+  let operator = getValidOperator(promptInput.operation);
+    if (operator == null) {
+      console.log(`Invalid operator: ${promptInput.operation}`);
+      return;
+    };
+
+    const addNums = (num1, num2) => { return`${num1} + ${num2} = ${num1 + num2}`};
```

Comment:
If you only have a single expression in an arrow function you can save yourself a little typing:

```suggestion
    const addNums = (num1, num2) => `${num1} + ${num2} = ${num1 + num2}`;
```